app/views/pg_hero/home/data_distribution.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="content">
+  <h1>Data Distribution</h1>
+  <h3>Shard Colocation Groups</h3>
+  <p><b><span style="color: #09436a;">NOTE: </span></b>The colocations include only shards of the tables that are <em>hash</em> distributed.</p>
+  <table class="table colocations-table">
+    <thead>
+      <tr>
+        <th style="width: 10%;"><%= link_to "Node ID", {} %></th>
+        <th style="width: 22%;"><%= link_to "Coloc. Group Size", {sort: "colocated_shards_size"} %></th>
+        <th colspan = "3">Shard Names</th>
+        <th>Shard Sizes</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @colocated_shard_sizes.each do |colocated_shards| %>
+        <% shard_names = colocated_shards[:shard_group].delete('{}').split(',') %>
+        <% shard_sizes = colocated_shards[:each_shard_size].delete('{}').split(',') %>
```

Comment:
why we need to delete {}?

Reply:
Otherwise they are kept in the name and size of the first and last appearing element.

Reply:
Like this
![screen shot 2018-09-14 at 10 42 10](https://user-images.githubusercontent.com/37271756/45536689-dce55180-b80a-11e8-9386-dc648f945512.png)